src/WorkflowCore/Interface/ICatchStepBuilder.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace WorkflowCore.Interface
+{
+    public interface ICatchStepBuilder<TData, TStepBody> : IStepBuilder<TData, TStepBody>, ITryStepBuilder<TData, TStepBody>
```

Comment:
Do we need an empty interface?

Reply:
This interface gives a builder full functionality of `IStepBuilder` and also allows to `Catch()` if we want to have multiple catches after one `Try()`. It is used as a return type for builder after the first `Catch()`, whereas after `Try()` we want to allow only to `Catch()`.